ansible_collections/arista/cvp/molecule/cv_configlet_v3/test_cv_configlet_v3.yaml

```diff
@@ -0,0 +1,93 @@
+---
+- name: Test cv_configlet_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_CONFIGLETS:
+      configlet1: 'alias a{{ 999 | random }} show lldp'
+      configlet2: 'alias c{{ 999 | random }} show version'
+  tasks:
+    - name: "Push config"
+      arista.cvp.cv_configlet_v3:
+        configlets: "{{CVP_CONFIGLETS}}"
+        state: present
+      register: CV_CONFIGLET_V3_RESULT
+
+    - name: "Check configlet creation process"
+      assert:
+        that:
+          - CV_CONFIGLET_V3_RESULT.changed == true
+          - CV_CONFIGLET_V3_RESULT.configlets_created.configlets_created_count == 2
+          - CV_CONFIGLET_V3_RESULT.configlets_created.configlets_created_list == ['configlet1', 'configlet2']
+          - CV_CONFIGLET_V3_RESULT.configlets_created.success == true
+
+    - name: "Push already existing config"
```

Comment:
This will change the config `configlet1` and `configlet2` since we are using `random` in line #8 and #9. This is a good test, but it's testing changed configuration. Update the name accordingly.
Also, We need to have another test where wee are pushing configlet with same configuration.

Reply:
fixed!